tests/test_anndata.py

```diff
@@ -129,9 +144,7 @@ def test_generate_metadata_file(self):
                 "library_preparation_protocol__ontology_label\n",
             ]
             self.assertEqual(
-                expected_line,
-                line,
-                'did not get expected headers from metadata body',
+                expected_line, line, 'did not get expected headers from metadata body'
```

Comment:
Does Black format this file?  IIUC, [Black would normally transform this to double quotes](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#strings).  As is, I can't discern why this string is single-quoted and strings above are a mix of single- and double-quoted, which is a bit distracting.

Reply:
Long ago, we wanted to use single quotes but Black uses double. We decided to use `--skip-string-normalization` so Black would ignore quote formatting. In retrospect, not a great idea.

I'm having a bit of bother with Black right now. Once that's addressed, I'd like to do a comprehesive switch to Black's double quote convention so that we have less mixed quote chaos. I'd rather not try to convert quotes in this PR for readability considerations.